themes/Snoonet/layouts/partials/git.html

```diff
@@ -0,0 +1,11 @@
+<em>
+    <a
+        href="{{ .Site.Params.repositoryURL }}commit/{{ .GitInfo.Hash }}"
+    >
+        {{ .GitInfo.AbbreviatedHash }}
+    </a>
+    &middot;
+    {{ .GitInfo.AuthorName }}
+    &middot;
+    {{ dateFormat "Mon, Jan 2, 2006 15:04:05" .GitInfo.AuthorDate }}
+</em>
```

Comment:
If you wrap this in `{{with .GitInfo}}` `{{end}}` it should make it so if the git info isn't available, it won't render.